toolchains/genoa.gcc.adastra.spack/environment.sh

```diff
@@ -0,0 +1,34 @@
+ENVIRONMENT_ROOT_DIRECTORY="$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &>/dev/null && pwd -P)"
```

Comment:
You may want to add these lines at the top of the file as a guard against misuse
https://github.com/gyselax/gyselalibxx/blob/7065cbaeaf0653c33fb763ed170641d22a6f8fe2/toolchains/v100.persee/environment.sh#L1-L6